api/src/main/resources/workbench.yaml

```diff
@@ -684,6 +685,29 @@ definitions:
         type: integer
         format: int64
         description: Milliseconds since the UNIX epoch.
+      UserRoles:
+        $ref: "#/definitions/UserRoleList"
+
+  UserRole:
+    type: object
+    required:
+      - user
+      - role
+    properties:
+      user:
+        type: string
+      role:
+        type: string
```

Comment:
Are roles free-form strings? I'd prefer enums unless we need arbitrary/runtime variation. Do we expect the set to grow -- or just owner/editor/viewer?

Reply:
+1 to using an enum in our Swagger definition.

Reply:
I think just owner/writer/reader. I'll move over to using an enum there.